DirectProgramming/C++SYCL_FPGA/Tutorials/Features/task_sequences/README.md

```diff
@@ -0,0 +1,250 @@
+>
+> For more information on configuring environment variables, see [Use the setvars Script with Linux* or macOS*](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup/use-the-setvars-script-with-linux-or-macos.html) or [Use the setvars Script with Windows*](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup/use-the-setvars-script-with-windows.html).
+
+### On Linux*
+
+1. Change to the sample directory.
+2. Build the program for Intel® Agilex® 7 device family, which is the default.
+   ```
+   mkdir build
+   cd build
+   cmake .. -DTYPE=<NAIVE/TASK_SEQUENCE>
+   ```
+   > **Note**: You can change the default target by using the command:
+   >  ```
+   >  cmake .. -DFPGA_DEVICE=<FPGA device family or FPGA part number> -DTYPE=<NAIVE/TASK_SEQUENCE>
+   >  ```
+
+3. Compile the design. (The provided targets match the recommended development flow.)
+
+   1. Compile for emulation (fast compile time, targets emulates an FPGA device).
+      ```
+      make fpga_emu
+      ```
+   2. Generate the HTML optimization reports.
+      ```
+      make report
+      ```
+      The report resides at `task_sequences.report.prj/reports/report.html`.
+   3. Compile for simulation (fast compile time, targets simulator FPGA device).
+      ```
+      make fpga_sim
+      ```
+   4. Compile with Quartus place and route (To get accurate area estimate, longer compile time).
+      ```
+      make fpga
+      ```
+
+### On Windows*
+
+1. Change to the sample directory.
+2. Configure the build system for the Intel® Agilex® 7 device family, which is the default.
+   ```
+   mkdir build
+   cd build
+   cmake -G "NMake Makefiles" .. -DTYPE=<NAIVE/TASK_SEQUENCE>
+   ```
+   > **Note**: You can change the default target by using the command:
+   >  ```
+   >  cmake -G "NMake Makefiles" .. -DFPGA_DEVICE=<FPGA device family or FPGA part number> -DTYPE=<NAIVE/TASK_SEQUENCE>
+   >  ```
+
+3. Compile the design. (The provided targets match the recommended development flow.)
+
+   1. Compile for emulation (fast compile time, targets emulated FPGA device).
+      ```
+      nmake fpga_emu
+      ```
+   2. Generate the optimization report. 
+      ```
+      nmake report
+      ```
+   3. Compile for simulation (fast compile time, targets simulator FPGA device).
+      ```
+      nmake fpga_sim
+      ```
+   4. Compile with Quartus place and route (To get accurate area estimate, longer compile time).
+      ```
+      nmake fpga
+      ```
+
+> **Note**: If you encounter any issues with long paths when compiling under Windows*, you may have to create your ‘build’ directory in a shorter path, for example `C:\samples\build`.  You can then run cmake from that directory, and provide cmake with the full path to your sample directory.
+
+## Run the Design
+
+### On Linux
```

Comment:
These sections are missing the .fpga run instructions

Reply:
If there is no BSP with host pipes, then there is no way for the user to generate a runnable fpga binary.